dll/win32/ieframe/CMakeLists.txt

```
@@ -14,6 +14,7 @@ list(APPEND SOURCE
     iehtmlwnd.c
     iexplore.c
     intshcut.c
+    inetfolder.c
```

keeping alphabetical order:
```suggestion
    inetfolder.c
    intshcut.c
```